frontend/src/pages/FAQ.vue

```diff
@@ -204,7 +204,12 @@
               >,
             </template>
             <template v-slot:two>
-              <a class="hyperlink" href="https://eprint.iacr.org/2021/089.pdf\" target="_blank">2</a>
+              <a
+                class="hyperlink"
+                href="https://web.archive.org/web/20221012070449/https://eprint.iacr.org/2021/089.pdf"
```

Comment:
just needed to remove the traling `\` 😅
```suggestion
                href="https://eprint.iacr.org/2021/089.pdf"
```

Reply:
Nice😂